### PR TITLE
BUGFIX: create a git dir unless it exist

### PR DESCRIPTION
**Pull request by MalloZup**
## What does this PR do?
FIx an issue before it becomes one 

## Tests written? 

No but https://github.com/openSUSE/gitarro/issues/76

### CODE REVIEW

**Comment by MalloZup** [comment]
@meaksh  have a look, but the unless is needed there

File: lib/gitarro/git_op.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def del_pr_branch(upstream, pr)
   def ck_or_clone_git
     git_repo_dir = git_dir + '/' + @options[:repo].split('/')[1]
     return if File.directory?(git_repo_dir)
-    FileUtils.mkdir_p(git_dir) if File.directory?(git_dir)
+    FileUtils.mkdir_p(git_dir) unless File.directory?(git_dir)
```
Comment (MalloZup) [inline lib/gitarro/git_op.rb]: we need unlss, otherwise we check create the dir if the dir exist

**Comment by MalloZup** [comment]
robo

File: lib/gitarro/git_op.rb (hunk 1 above)
Comment (meaksh) [reply lib/gitarro/git_op.rb]: Oops, that's totally true @MalloZup! I didn't catch it! Thanks for fixing! 🚀

**Pull request merged**